### PR TITLE
Render-aig

### PR DESCRIPTION
**Pull request by daisyfaithauma**
- Rendered 'create gateway' form the partial "../../partials/ai-gateway/create-gateway"

### CODE REVIEW

File: src/content/docs/ai-gateway/providers/perplexity.mdx [hunk 1]
```diff
@@ -30,13 +29,15 @@ curl https://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_id}/perplexity-a
...
 
-```js title="JavaScript"
+````js title="JavaScript"
 import OpenAI from "openai";
 
 const apiKey = env.PERPLEXITY_API_KEY;
 const accountId = '{account_id}';
 const gatewayId = '{gateway_id}';
 const baseURL = `https://gateway.ai.cloudflare.com/v1/${accountId}/${gatewayId}/perplexity-ai`;
+```javascript
```
Comment (kodster28) [inline src/content/docs/ai-gateway/providers/perplexity.mdx]: This looks like a merge conflict that was addressed incorrectly?

File: src/content/docs/ai-gateway/providers/perplexity.mdx [hunk 2]
```diff
@@ -30,13 +29,15 @@ curl https://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_id}/perplexity-a
 
 Perplexity doesn't have their own SDK, but they have compatibility with the OpenAI SDK. You can use the OpenAI SDK to make a Perplexity call through AI Gateway as follows:
 
-```js title="JavaScript"
+````js title="JavaScript"
```
Comment (kodster28) [inline src/content/docs/ai-gateway/providers/perplexity.mdx]:
```suggestion
```js title="JavaScript"
```

File: src/content/docs/ai-gateway/providers/perplexity.mdx [hunk 3]
```diff
@@ -53,4 +54,4 @@ const chatCompletion = await perplexity.chat.completions.create({
   max_tokens: maxTokens,
 });
 
-```
+````
```
Comment (kodster28) [inline src/content/docs/ai-gateway/providers/perplexity.mdx]: Move to one `

**Review by kodster28 — CHANGES_REQUESTED** [review]

File: src/content/docs/ai-gateway/providers/azureopenai.mdx [hunk 4]
```diff
@@ -45,19 +44,25 @@ If you are using the `openai-node` library, you can set your endpoint like this:
...
-  baseURL,
-  defaultQuery: { 'api-version': apiVersion },
-  defaultHeaders: { 'api-key': apiKey },
+	apiKey,
+	baseURL,
+	defaultQuery: { "api-version": apiVersion },
+	defaultHeaders: { "api-key": apiKey },
 });
 
+const azure_openai = new OpenAI({
```
Comment (pedrosousa) [inline src/content/docs/ai-gateway/providers/azureopenai.mdx]: We seem to be adding `const azure_openai = (...)` for the second time here... can you check if this is duplicated?

File: src/content/docs/ai-gateway/providers/huggingface.mdx [hunk 5]
```diff
@@ -39,5 +38,8 @@ const apiToken = env.HF_API_TOKEN;
 const hf = new HfInferenceEndpoint(
   baseURL,
   apiToken
+  baseURL,
+  apiToken
```
Comment (pedrosousa) [inline src/content/docs/ai-gateway/providers/huggingface.mdx]: I think we're also duplicating these two lines.

File: src/content/docs/ai-gateway/providers/perplexity.mdx [hunk 6]
```diff
@@ -34,23 +33,24 @@ Perplexity doesn't have their own SDK, but they have compatibility with the Open
 import OpenAI from "openai";
 
 const apiKey = env.PERPLEXITY_API_KEY;
-const accountId = '{account_id}';
-const gatewayId = '{gateway_id}';
+const accountId = "{account_id}";
+const gatewayId = "{gateway_id}";
 const baseURL = `https://gateway.ai.cloudflare.com/v1/${accountId}/${gatewayId}/perplexity-ai`;
 
+import OpenAI from "openai";
```
Comment (pedrosousa) [inline src/content/docs/ai-gateway/providers/perplexity.mdx]: This `import` is already in line 33 above. Can you check if it's needed?

File: src/content/docs/ai-gateway/tutorials/create-first-aig-workers.mdx [hunk 7]
```diff
@@ -53,25 +43,18 @@ This tutorial guides you through creating your first AI Gateway using Workers AI
 The expected output would be similar to :
 
 ```bash
```
Comment (pedrosousa) [inline src/content/docs/ai-gateway/tutorials/create-first-aig-workers.mdx:45]:
```suggestion
```json
```

File: src/content/docs/ai-gateway/tutorials/create-first-aig-workers.mdx [hunk 8]
```diff
@@ -53,25 +43,18 @@ This tutorial guides you through creating your first AI Gateway using Workers AI
...
...ring your site remains accessible.\n3. **Firewall**: Cloudflare acts as an additional layer of security, filtering out malicious traffic and hacking attempts, such as SQL injection or cross-site scripting (XSS) attacks.\n4. **SSL Encryption**: Cloudflare offers free SSL encryption, which secure sensitive information (like passwords, credit card numbers, and browsing data) with an HTTPS connection (the \"S\" stands for Secure).\n5. **Bot Protection**: Cloudflare has an AI-driven system that identifies and blocks bots trying to exploit vulnerabilities or scrape your content.\n6. **Analytics**: Cloudflare provides insights into website traffic, helping you understand your audience and make informed decisions.\n7. **Cybersecurity**: Cloudflare offers advanced security features, such as intrusion protection, DNS filtering, and Web Application Firewall (WAF) protection.\n\nOverall, Cloudflare helps protect against cyber threats, improves website performance, and enhances security for online businesses, bloggers, and individuals who need to establish a strong online presence.\n\nWould you like to know more about a specific aspect of Cloudflare?"},"success":true,"errors":[],"messages":[]}%
```
Comment (pedrosousa) [inline src/content/docs/ai-gateway/tutorials/create-first-aig-workers.mdx:46]:
```suggestion
{"result":{"response":"I'd be happy to explain what Cloudflare is.\n\nCloudflare is a cloud-based service that provides a range of features to help protect and improve the performance, security, and reliability of websites, applications, and other online services. Think of it as a shield for your online presence!\n\nHere are some of the key things Cloudflare does:\n\n1. **Content Delivery Network (CDN)**: Cloudflare has a network of servers all over the world. When you visit a website that uses Cloudflare, your request is sent to the nearest server, which caches a copy of the website's content. This reduces the time it takes for the content to load, making your browsing experience faster.\n2. **DDoS Protection**: Cloudflare protects against Distributed Denial-of-Service (DDoS) attacks. This happens when a website is overwhelmed with traffic from multiple sources to make it unavailable. Cloudflare filters out this traffic, ensuring your site remains accessible.\n3. **Firewall**: Cloudflare acts as an additional layer of security, filtering out malicious traffic and hacking attempts, such as SQL injection or cross-site scripting (XSS) attacks.\n4. **SSL Encryption**: Cloudflare offers free SSL encryption, which secure sensitive information (like passwords, credit card numbers, and browsing data) with an HTTPS connection (the \"S\" stands for Secure).\n5. **Bot Protection**: Cloudflare has an AI-driven system that identifies and blocks bots trying to exploit vulnerabilities or scrape your content.\n6. **Analytics**: Cloudflare provides insights into website traffic, helping you understand your audience and make informed decisions.\n7. **Cybersecurity**: Cloudflare offers advanced security features, such as intrusion protection, DNS filtering, and Web Application Firewall (WAF) protection.\n\nOverall, Cloudflare helps protect against cyber threats, improves website performance, and enhances security for online businesses, bloggers, and individuals who need to establish a strong online presence.\n\nWould you like to know more about a specific aspect of Cloudflare?"},"success":true,"errors":[],"messages":[]}
```
I think there's an extra `%` at the end. Can you check?

**Review by pedrosousa — APPROVED** [review]

File: src/content/docs/ai-gateway/providers/huggingface.mdx (hunk 5 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/providers/huggingface.mdx]: I think these duplicates are coming from a merge conflict I had. Thanks.

File: src/content/docs/ai-gateway/tutorials/create-first-aig-workers.mdx (hunk 8 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/tutorials/create-first-aig-workers.mdx:46]: Yes

**Review by kodster28 — APPROVED** [review]
tthough you have a merge conflict

**Pull request merged**